.release-it.json

```diff
@@ -0,0 +1,17 @@
+{
+  "git": {
+    "requireBranch": "master",
+    "commitMessage": "Version ${version}",
+    "tagName": "${version}",
+    "tagAnnotation": "Version ${version}"
```

Comment:
Where is this version coming? Should we move this to a Grunt task?

Reply:
Thanks, @kasparsd.

I explored the option of going with the Grunt task — based on the example we saw —, but the tool evolved, and some of the methods are no longer available. 

I've added a new dependency that allows reading the version from a file.
Since we already had logic to update the `readme.txt` in a grunt task, I thought we could use a similar approach to update the `cloudinary.php`.

What do you think?